src/main/java/io/debezium/connector/jdbc/HibernateJAXBRecordMapper.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.jdbc;
+
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.util.Locale;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+
+import org.apache.kafka.connect.data.Field;
+import org.apache.kafka.connect.data.Schema;
+import org.hibernate.boot.jaxb.hbm.spi.JaxbHbmBasicAttributeType;
+import org.hibernate.boot.jaxb.hbm.spi.JaxbHbmHibernateMapping;
+import org.hibernate.boot.jaxb.hbm.spi.JaxbHbmIdentifierGeneratorDefinitionType;
+import org.hibernate.boot.jaxb.hbm.spi.JaxbHbmRootEntityType;
+import org.hibernate.boot.jaxb.hbm.spi.JaxbHbmSimpleIdType;
+import org.hibernate.type.BigIntegerType;
+import org.hibernate.type.BooleanType;
+import org.hibernate.type.ByteType;
+import org.hibernate.type.DateType;
+import org.hibernate.type.FloatType;
+import org.hibernate.type.IntegerType;
+import org.hibernate.type.LongType;
+import org.hibernate.type.TextType;
+import org.hibernate.type.TimeType;
+import org.hibernate.type.TimestampType;
+
+import io.debezium.data.Bits;
+import io.debezium.time.Date;
+import io.debezium.time.MicroDuration;
+import io.debezium.time.ZonedTime;
+import io.debezium.time.ZonedTimestamp;
+
+/**
+ *
+ */
+public class HibernateJAXBRecordMapper {
```

Comment:
Could you add a class-level JavaDoc comment describing its purpose (should be the case for every class. Also "JAXB" -> "Jaxb" (as per the general naming convention in the code base when it comes to abbreviations in identifiers).